### PR TITLE
Update OLM package for 4.3.1

### PR DESCRIPTION
**Pull request by cbandy**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

OLM does not yet support the creation of ConfigMaps, Secrets, or Services, so most of this work is updating the instructions for creating those objects. Quite a few steps have been eliminated by changing  `install-bootstrap-creds.sh` and `client-setup.sh` to work with `oc` and `kubectl` and without `expenv`. The objects that OLM does create are updated in `postgresoperator.csv.yaml`.

The Kubernetes and OpenShift instructions are largely the same with two notable exceptions:
1. OpenShift includes a step to work in the `restricted` SCC, and
2. OpenShift describes how to use a Route to expose the PostgreSQL Operator API, as recommended in #1464.

### CODE REVIEW

File: installers/olm/openshift.description.md [hunk 1]
```diff
@@ -34,166 +34,112 @@ and much more!
...
+Install the Security Context Constraint for OpenShift:
 
 ```
-storageos:
-  AccessMode:  ReadWriteOnce
-  Size:  1G
-  StorageType:  dynamic
-  StorageClass:  gp2
-  Fsgroup:  26
+oc create -f ./deploy/pgo-scc.yaml
```
Comment (jkatz) [inline installers/olm/openshift.description.md:63]: Perhaps due to the hubbub with SCCs lately, we should include a note around using this vs. running in restricted mode?

File: installers/olm/openshift.description.md [hunk 2]
```diff
@@ -34,166 +34,112 @@ and much more!
...
-```
-PrimaryStorage: storageos
-ReplicaStorage: storageos
-BackrestStorage: storageos
-```
+### PostgreSQL Operator Configuration
 
-### Certificates
+Edit `conf/postgres-operator/pgo.yaml` to configure the deployment. Look over all of the options and make any
+changes necessary for your environment. A [full description of each option][pgo-yaml-reference] is available in the documentation.
```
Comment (jkatz) [inline installers/olm/openshift.description.md:69]: With #1519 we should also include "this is what you edit in here if you want to run in restricted mode" for SCC

File: installers/olm/openshift.description.md (hunk 1 above)
Comment (cbandy) [reply installers/olm/openshift.description.md]: Replaced with `DisableFSGroup: true`.

File: installers/olm/openshift.description.md (hunk 2 above)
Comment (cbandy) [reply installers/olm/openshift.description.md:68]: Done.

**Pull request merged**